### PR TITLE
[DOCS] EQL: Document `concat` function

### PR DESCRIPTION
**Pull request by jrodewig**
Relates to #55193 and #55185

### CODE REVIEW

File: docs/reference/eql/functions.asciidoc [hunk 1]
```diff
@@ -180,6 +181,57 @@ CIDR block you wish to search. If `null`, the function returns `null`.
...
+[source,txt]
+----
+concat(<value>[, <value>])
+----
+
+*Parameters*
+
+`<value>`::
+(Required{multi-arg-ref})
+Value to concatenate. If `null`, the function returns `null`.
```
Comment (rw-access) [inline docs/reference/eql/functions.asciidoc:227]: I don't have a strong grasp on the verbiage for `multi-arg-ref`
```suggestion
Value to concatenate. If any of the arguments are `null`, the function returns `null`.
```

**Review by rw-access — APPROVED** [review]

File: docs/reference/eql/functions.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/eql/functions.asciidoc]: I think this clarification is good regardless. Thanks for adding.

The `{multi-arg-ref}` attribute creates a footnote link to the following text:

> This parameter accepts multiple arguments.

Here's a screenshot from the `wildcard` function, which happens to be near the footnote at the bottom of the page.

Screen Shot 2020-05-05 at 4 24 47 PM

**Comment by jrodewig** [comment]
Thanks @rw-access

**Comment by jrodewig** [comment]
### Backport commits
master c7ac7e0
7.x 2ae24c7c093

**Pull request merged**